Api Web/Models/Department.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Api_Web.Models
+{
+    public class Department
+    {
+        [Key]
+        public int DepartmentEmployeeId { get; set; }
+        public string Departments{ get; set; }
```

Comment:
If we say departments, we expect to have many, but the variable is one single string, what do we expect to receive here?